website/releases/release-0.15.0.md

```diff
@@ -0,0 +1,339 @@
+options should be set with a unique value, a timestamp value or UUID as recommended. Setting this config indicates that
+the subsequent sync should ignore the last committed checkpoint for the source. The config value is stored in the commit
+history, so setting the config with same values would not have any affect. This config can be used in scenarios like
+kafka topic change where we would want to start ingesting from the latest or earliest offset after switching the topic
+(in this case we would want to ignore the previously committed checkpoint, and rely on other configs to pick the
+starting offset).
+
+### Meta Sync Improvements
+
+#### Paralleled Listing in Glue Catalog Sync
+
+AWS Glue Catalog sync now supports paralleled listing of partitions to improve the listing performance and reduce meta
+sync latency. Three new configs are added to control the listing parallelism:
+
+- [`hoodie.datasource.meta.sync.glue.all_partitions_read_parallelism`](/docs/configurations#hoodiedatasourcemetasyncglueall_partitions_read_parallelism):
+  parallelism for listing all partitions (first time sync).
+- [`hoodie.datasource.meta.sync.glue.changed_partitions_read_parallelism`](/docs/configurations#hoodiedatasourcemetasyncgluechanged_partitions_read_parallelism):
+  parallelism for listing changed partitions (second and subsequent syncs).
+- [`hoodie.datasource.meta.sync.glue.partition_change_parallelism`](/docs/configurations#hoodiedatasourcemetasyncgluepartition_change_parallelism):
+  parallelism for change operations such as create, update, and delete.
+
+#### BigQuery Sync Optimization with Metadata Table
+
+The BigQuery Sync now loads all partitions from the metadata table once if the metadata table is enabled, to improve the
+file listing performance.
+
+### Metrics Reporting to M3
+
+A new MetricsReporter
+implementation [`M3MetricsReporter`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/metrics/m3/M3MetricsReporter.java)
+is added to support emitting metrics to [M3](https://m3db.io/). Users can now enable reporting metrics to M3 by
+setting [`hoodie.metrics.reporter.type`](/docs/basic_configurations#hoodiemetricsreportertype) as `M3` and their
+corresponding
+host address and port in [`hoodie.metrics.m3.host`](/docs/basic_configurations#hoodiemetricsreportertype)
+and [`hoodie.metrics.m3.port`](/docs/basic_configurations#hoodiemetricsm3port).
+
+### Other Features, Improvements and Bug Fixes
+
+#### Schema Exception Classification
+
+This release introduces the classification of schema-related
+exceptions ([HUDI-7486](https://issues.apache.org/jira/browse/HUDI-7486)) to make it easy for users to
+understand the root cause, including the errors during converting the records from Avro to Spark Row due to illegal
+schema, or the records are incompatible with the provided schema.
+
+#### Index Lookup Duration Metric
+
+This release fixes the index lookup duration metric in Spark by accounting for the lazy
```

Comment:
these are minor. lets not bloat release highlights w/ every change. we should try to keep release page succint calling out major features. if you check 0.8.0 or 0.9.0, we tried to keep it very very brief and not be very verbose.

Reply:
I included these three bug fixes particularly since they have a fairly large impact radius, and users can notice the difference after upgrading the 0.15.0 release.  So it makes sense to include them.  They are folded under the section `Other Features, Improvements and Bug Fixes` so the release notes are well structured.

Reply:
Removed.